packages/webapp/src/containers/Finances/CustomRevenueType/EditCustomRevenue.jsx

```diff
@@ -34,7 +33,7 @@ function EditCustomExpense({ history, match }) {
   };
 
   const onSubmit = (payload) => {
-    dispatch(updateCustomRevenueType(payload, revenue_type_id));
+    dispatch(updateCustomRevenueType({ ...payload, revenue_type_id: Number(revenue_type_id) }));
```

Comment:
You don't need to convert it for the sagas / API, I think because it will be sent as a parameter.

Reply:
I can admit to not having a reason I did this before haha

But the putTaskEnqueSuccess would use the id to update state so I think it is good. Same with the deleteTaskSuccess

Reply:
Ah, true.